input/input.go

```diff
@@ -19,24 +22,42 @@ type model struct {
 	textinput   textinput.Model
 	quitting    bool
 	aborted     bool
+	timeout     time.Duration
+	hasTimeout  bool
 }
 
-func (m model) Init() tea.Cmd { return textinput.Blink }
+func (m model) Init() tea.Cmd {
+	return tea.Batch(
+		textinput.Blink,
+		timeout.Init(m.timeout, nil),
+	)
+}
 func (m model) View() string {
 	if m.quitting {
 		return ""
 	}
-
+	var timeStr string
+	if m.hasTimeout {
+		timeStr = timeout.Str(m.timeout)
+	}
 	if m.header != "" {
 		header := m.headerStyle.Render(m.header)
-		return lipgloss.JoinVertical(lipgloss.Left, header, m.textinput.View())
+		return lipgloss.JoinVertical(lipgloss.Left, header, m.textinput.View()+" "+timeStr)
 	}
 
-	return m.textinput.View()
+	return timeStr + " " + m.textinput.View()
```

Comment:
IMO we should remove this time indicator as it is not consistent and not customizable, we can add a customizable timer later!

Reply:
Removing it for input only or always ? I would like to see them for confirm for example. But you are right, ist not styleable nor customizable.

Reply:
`gum confirm` is okay since it's a bit of a special case but just remove it for `gum input` imo since it seems a bit out of place